tests/test_inline.py

```diff
@@ -0,0 +1,134 @@
+import traceback
+from unittest import TestCase
```

Comment:
I'm happy to convert my tests to `pytest` (whatever that is :)). I only used `unittest` because that's what the pycharm tutorial used! Something else for me to learn.

Reply:
Converted to pytest